.rubocop_todo.yml

```diff
@@ -0,0 +1,187 @@
+# This configuration was generated by
```

Comment:
This isn't optional, this is so we don't have to fix all the linting errors now.